alerter.cpp

```diff
@@ -1,32 +1,47 @@
 #include <iostream>
 #include <assert.h>
 
+
 int alertFailureCount = 0;
+int temperature_thresh= 200.0;
 
 int networkAlertStub(float celcius) {
     std::cout << "ALERT: Temperature is " << celcius << " celcius.\n";
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+    if(celcius>temperature_thresh)

```

Comment:
the intention of this session is to describe the behavior via test /asserts. Fixing the code or adding implementation should not be done.

Assume boundary conditions when the values should be treated as within or out of limits, and then assert them in test code. Do not update the code. let the tests fail!!